docker-compose/agent.yaml

```diff
@@ -25,36 +29,37 @@ prometheus:
               regex: '.*'
               target_label: instance
               replacement: ${HOST}
-      remote_write:
-        - url: ${OPSVERSE_METRICS_ENDPOINT}
-          basic_auth:
-            username: ${OPSVERSE_USERNAME}
-            password: ${OPSVERSE_PASSWORD}
-loki:
-  positions_directory: /tmp/
-  configs:
-    - name: local
-      clients:
-        - url: ${OPSVERSE_LOGS_ENDPOINT}
-          basic_auth:
-            username: ${OPSVERSE_USERNAME}
-            password: ${OPSVERSE_PASSWORD}
-      scrape_configs:
-        # Add a job for any new dir that needs to be tailed for logs
-        - job_name: var-log
-          static_configs:
-            - labels:
-                job: varlog
-                host: ${HOST}
-                __path__: /var/log/*.log
-        - job_name: containers
```

Comment:
We do not need this section as the `docker_sd` should scrape the docker logs.

Reply:
Made the necessary changes.
Commit: https://github.com/OpsVerseIO/installers/pull/23/commits/d3020f2a8fcbc6ebf11c4cfc65d9269160059f30